packages/core/src/app/address/googleAutocomplete/AddressSelector.ts

```diff
@@ -22,7 +22,7 @@ export default class AddressSelector {
     }
 
     getStreet2(): string {
-        return '';
+        return this._get('subpremise', 'short_name');
```

Comment:
It turns out that Google Map API 's defintion of `subpremise` is not always constant. Sometimes, unit number is not a `subpremise` type. But this should be the best we could do.

Reply:
yes we just have to address what's returned for the value.